docs/DeveloperGuide.adoc

```diff
@@ -424,6 +424,56 @@ Given below is the Sequence Diagram for the interactions with the `Logic` compon
 image::SortSequenceDiagram.png[width=790]
 +
 
+=== Use case: List all patients/donors/doctors
+
+Given below is the activity diagram to illustrate what happens when user uses the list command.
+
+image::ListActivityDiagram.png[]
+
+*MSS*
+
+1.  User requests to list patients/donors/doctors.
+2.  ORGANice shows the list of patients/donors/doctors.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. ORGANice detects an invalid parameter.
++
+[none]
+** 1a1. ORGANice requests for correct parameter.
++
+** 1a2. User enters correct parameter.
++
+Steps 1a1-1a2 are repeated until the data entered are correct.
++
+Use case resumes at step 2.
+
+=== Use case: Sort
+
+*MSS*
+
+1.  User requests to sort the list of donors of a particular patient.
+2.  ORGANice sorts the list of donors according to the parameters.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. ORGANice detects invalid parameter(s).
++
+[none]
+** 1a1. ORGANice prompts for valid parameter(s).
++
+** 1a2. User enters correct parameter(s).
++
+Steps 1a1-1a2 are repeated until the data entered are correct.
++
+Use case resumes at step 2.
+
```

Comment:
Not sure why it is not listed here, but the line below Figure 9 is supposed to be a title, and the markdown is not showing it as a title. But it is shown as this: "=== Use case: List all patients/donors/doctors"

Reply:
Fixed.